src/modules/admin/components/Profile/ColonyAvatarUploader.tsx

```diff
@@ -53,6 +53,7 @@ const ColonyAvatarUploader = ({ colony: { colonyAddress }, colony }: Props) => {
     <AvatarUploader
```

Comment:
When uploading an image that is not supported (in this case too large):
![Screenshot from 2021-01-19 13-50-00](https://user-images.githubusercontent.com/1193222/105032537-8eda5b80-5a5f-11eb-9ac9-fad933b2d959.png)

It triggers some sort of error in `AvatarUploadItem`:
![Screenshot from 2021-01-19 14-07-09](https://user-images.githubusercontent.com/1193222/105032595-a6194900-5a5f-11eb-9743-6bf5091c4739.png)

I know this is not something you introduced, but please investigate its cause. (To me it looks like a bad import)

Reply:
i don't have this error - i just see an error UI - as expected

<img width="1268" alt="Screenshot 2021-01-19 at 20 50 15" src="https://user-images.githubusercontent.com/34057551/105091968-894d3780-5a98-11eb-8b53-848cb404a756.png">

Reply:
this is both for larger & different format errors... not sure how i can replicate.

Reply:
Bah. Was this on my machine only? I'll have to check again I guess...